tools/clang/test/HLSL/system-values.hlsl

```diff
@@ -40,6 +40,8 @@
 #define Def_Barycentrics DECLARE(float3 BaryWeights : SV_Barycentrics) USE(float, BaryWeights.x) USE(float, BaryWeights.y) USE(float, BaryWeights.z)
 #define Def_ShadingRate DECLARE(uint rate : SV_ShadingRate) USE(uint, rate)
 #define Def_CullPrimitive DECLARE(bool cullprim : SV_CullPrimitive) USE(bool, cullprim)
+#define Def_BaseVertexLocation DECLARE(uint svloc : SV_BaseVertexLocation) USE(uint, svloc)
```

Comment:
Should the type for `SV_BaseVertexLocation` be `int` insetead of `uint`?

Reply:
Resolving this because it was at an outdated location, and I have a new comment.